lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/YarnClusterSubmissionFromCS.java

```diff
@@ -83,6 +84,7 @@ private YarnClusterSubmissionFromCS(final AvroYarnAppSubmissionParameters yarnAp
     this.queue = DEFAULT_QUEUE;
     this.tokenKind = yarnClusterJobSubmissionParameters.getSecurityTokenKind().toString();
     this.tokenService = yarnClusterJobSubmissionParameters.getSecurityTokenService().toString();
+    this.fileSystemUrl = yarnJobSubmissionParameters.getFileSystemUrl().toString();
```

Comment:
Why do we need that `toString()` call? isn't the result of `.getFileSystemUrl()` a string already?